api/gatewayserver.proto

```diff
@@ -49,6 +50,18 @@ service GtwGs {
   rpc LinkGateway(stream GatewayUp) returns (stream GatewayDown);
   // GetConcentratorConfig associated to the gateway.
   rpc GetConcentratorConfig(google.protobuf.Empty) returns (ConcentratorConfig);
+  // GetMQTTConnectionInfo returns the MQTT server address and the username for the gateway.
+  rpc GetMQTTConnectionInfo(GatewayIdentifiers) returns (MQTTConnectionInfo) {
+    option (google.api.http) = {
+      get: "/gs/gateways/{gateway_id}/mqtt-connection-info"
+    };
+  };
+  // GetMQTTv2ConnectionInfo returns the MQTT server address and the username for the gateway.
+  rpc GetMQTTv2ConnectionInfo(GatewayIdentifiers) returns (MQTTConnectionInfo) {
```

Comment:
"MQTT v2" may be a bit confusing, as it sounds like we're referring to an MQTT version. Not sure how to improve that...

Reply:
Not sure either; in GS config we also call this `mqtt-v2`, so it's consistent with that.

I'd say `GetMQTTV2ConnectionInfo()` though

Also for the comments, don't do not have to follow Go comment conventions. We used to do that, but we don't do that anymore because it doesn't look nice in generated docs.

Reply:
It follows the same convention as the GS configuration. I'll reword the comment.